cephfs/cephfs_test.go

```diff
@@ -15,14 +16,60 @@ import (
 )
 
 var (
-	CephMountTest = "/tmp/ceph/mds/mnt/"
+	CephMountDir     = "/tmp/ceph/mds/mnt/"
+	requireCephMount = false
+	testMdsName      = "Z"
 )
 
+func init() {
+	mdir := os.Getenv("GO_CEPH_TEST_MOUNT_DIR")
+	if mdir != "" {
+		CephMountDir = mdir
+	}
+	reqMount := os.Getenv("GO_CEPH_TEST_REQUIRE_MOUNT")
+	if reqMount == "yes" || reqMount == "true" {
+		requireCephMount = true
+	}
+	mdsName := os.Getenv("GO_CEPH_TEST_MDS_NAME")
+	if mdsName != "" {
+		testMdsName = mdsName
+	}
+}
+
+func useMount(t *testing.T) {
+	fail := func(m string) {
+		if requireCephMount {
+			t.Fatalf("cephfs mount required: %s %s", CephMountDir, m)
+		} else {
+			t.Skipf("cephfs mount needed: %s %s", CephMountDir, m)
+		}
+	}
+
+	s, err := os.Stat(CephMountDir)
+	if err != nil || !s.IsDir() {
+		fail("missing or not a directory")
```

Comment:
No evaluation of `err`?

Reply:
If err is set than we couldn't stat it for some reason or another, and would be invalid for the test anyway. Since this is test code I didn't try to make it extra robust.